src/pudl/output/pudltabl.py

```diff
@@ -145,21 +145,15 @@ def pu_eia860(self, update=False):
             )
         return self._dfs["pu_eia"]
 
-    def pu_ferc1(self, update=False):
+    def pu_ferc1(self):
         """Pull a dataframe of FERC plant-utility associations.
 
-        Args:
-            update (bool): If true, re-calculate the output dataframe, even if
-                a cached version exists.
-
         Returns:
             pandas.DataFrame: a denormalized table for interactive use.
```

Comment:
I've been trying to make sure I add type hints on all these methods as I update them too.